packages/dev/biome-config/configs/biome-package.jsonc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
can you mention new preset in the documentation?

Reply:
On it, thank you!

Reply:
btw, is the following correct:
```Backend:

```json
{
	"$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
	"extends": ["./node_modules/@lokalise/biome-config/configs/biome-base.jsonc", "./node_modules/@lokalise/biome-config/configs/biome-esm.jsonc"]
}
```

I think we are not using esm config, right? asking just to understand it 🙏

Reply:
newer services (Billing, Workflows) use ESM, Autopilot doesn't (benefits of a switch do not currently justify the effort needed)